mathics/eval/makeboxes/makeboxes.py

```diff
@@ -136,6 +137,19 @@ def int_to_string_shorter_repr(value: int, form: Symbol, max_digits=640):
     return String(value_str)
 
 
+def eval_makeboxes_outputform(expr, evaluation, form):
+    """
+    Build a 2D text representation of the expression.
```

Comment:
Something to 
```suggestion
    Build a 2D representation of the expression using only keyboard characters.
```

Something that's been slightly annoying that I'd like to mention. In naming, there has been this tendency to focus on 1D or 2D, but the focus in `OutputForm` is _keyboard characters only_.  

https://reference.wolfram.com/language/ref/OutputForm.html?q=OutputForm as usual gets this correct, but it is easy from that description to misunderstand the focus.